src/redux/books/Books.js

```diff
@@ -0,0 +1,40 @@
+const ADD_BOOKS = "ADD_BOOKS"
```

Comment:
- [ ] It would be better if our action types follow the `ducks` patthern. For example; `const ADD_BOOK = bookstore/books/ADD_BOOK` or in your case `const ADD_BOOK = "bookstore/books/ADD_BOOK"`